inc/sub-headerException.php

```diff
@@ -0,0 +1,124 @@
+<div class="newsSubHeader clearfix">
```

Comment:
Can you also put a note about what this file is for? I'm not clear from the file name...